components/Collections/examples/CollectionsEditingExample.m

```diff
@@ -64,9 +64,11 @@ - (void)updatedRightBarButtonItem:(BOOL)isEditing {
 }
 
 - (void)toggleEditMode:(id)sender {
-  BOOL isEditing = self.editor.isEditing;
-  [self updatedRightBarButtonItem:!isEditing];
-  [self.editor setEditing:!isEditing animated:YES];
+  if ([sender isEqual:self.navigationItem.rightBarButtonItem]) {
```

Comment:
This seems strange to me.  We are querying the sender, but it makes not difference in the body of the if clause.  If another control were to use this action if would fail, which would be surprising.
If we don't care what the sender it, drop that parameter.
Here and below.

Reply:
Agreed that I'm not certain how important it is for us to be checking the sender in these methods. If the intent was simply to make use of the sender then I would encourage us to simply mark the sender as `__unused` instead and keep the simpler logic we had before.

Reply:
ok im going to drop the sender.
For other cases that require the parameters I will be putting an NSLog statement to silence the warning.

Reply:
Nooooo. Mark the parameter as __unused.

Reply:
Why is `NSLog` preferred over `__unused`?  Can't you at least do something less verbose, like:
```objectivec
(void)unusedVariable;
```

Reply:
We are intentionally trying to keep the examples 'simple'
Beginning engineers will likely know what NSLog() is.  They are unlikely to be as familiar with __unused.